src/models/userDao.js

```diff
@@ -0,0 +1,144 @@
+const { AppDataSource } = require('./dataSource');
+
+const findUserById = async (id) => {
+  const [user] = await AppDataSource.query(
+    `
+    SELECT
+      seller_id AS sellerId
+    FROM
+      users
+    WHERE
+      id = ?
+    `,
+    [id]
+  );
+
+  return user;
+};
+
+const findUserBySellerId = async (sellerId) => {
+  const [user] = await AppDataSource.query(
+    `
+      SELECT
+        *
+      FROM
+        users
+      WHERE
+        seller_id = ?
+      LMIT 1
+    `,
+    [sellerId]
+  );
```

Comment:
```suggestion
const findUserBySellerId = async (sellerId) => {
  const [user] = await AppDataSource.query(
    `
      SELECT
        *
      FROM
        users
      WHERE
        seller_id = ?
      LMIT 1
    `,
    [sellerId]
  );
```
`*` 여기도 * 제외해 주세요 :)

Reply:
지금 쓰일 일이 없을 것 같아서 그냥 제거하겠습니당!